core/domain/state_domain.py

```diff
@@ -4073,7 +4089,9 @@ def from_dict(cls, state_dict: StateDict) -> State:
 
     @classmethod
     def create_default_state(
-        cls, default_dest_state_name: str, is_initial_state: bool = False
+        cls,
+        default_dest_state_name: Optional[str],
```

Comment:
Don't forget to update the docstring below.

Reply:
Done!